pandas/tseries/resample.py

```diff
@@ -208,6 +208,12 @@ def _assure_grouper(self):
         """ make sure that we are creating our binner & grouper """
         self._set_binner()
 
+    def plot(self, *args, **kwargs):
+        # for compat with prior versions, we want to
+        # have the warnings shown here and just have this work
+        return _maybe_process_deprecations(self, how='mean').plot(*args,
```

Comment:
Small remark, by passing `how='mean'` here, the user gets the message "FutureWarning: how in .resample() is deprecated", while he did not use 'how' (in eg ``s.resample('15min').plot()``)